app/pages/3_Country_operation.py

```diff
@@ -190,12 +190,16 @@ def get_colors_map():
 load_buses_space_heat_aggr["space heating original"] = load_buses_space_heat_aggr.sum(axis=1)
 load_buses_space_heat_aggr["space heating overall"] = load_buses_space_heat_aggr["space heating original"] - gen_buses_retrof_aggr.sum(axis=1)
 
+heat_techs = ["residential rural heat", "residential urban decentral heat",
+              "services rural heat", "services urban decentral heat",
+              "urban central heat"]
+
 with balance_plot_col:
     buses_heat_area_plot = load_buses_space_heat_aggr[load_buses_space_heat_aggr.columns.difference(["space heating overall", "space heating original"])].hvplot.area(
         **kwargs,
         ylabel="Heat Demand [MW]",
         group_label=helper.config["loads_t_parameter"]["p"]["legend_title"],
-        color = ["#ffc100", "#ff9a00", "#ff7400", "#ff4d00", "#ff0000"]
+        color = [tech_colors[x] for x in heat_techs]
```

Comment:
might be good to have this more general in case the order of heat_techs changes, in `load_buses_space_heat_aggr`. can heat_techs be somehow extracted from `load_buses_space_heat_aggr`?

if not, lets keep it like that. we have many hard coded elements. :)

Reply:
As a comment, `hvplot` is smart enough to re-cycle the colours, so that is safe to have there an arbitrary number of the columns. `heat techs` are hard-coded here to attract an attention to the need to position the data exactly in the order as it's shown on the picture. For some reasons, the visual impression strongly depends on the order of the colors.

I think, It's better to focus on the colors for this PR and adjust the data part as the next step. (Happy to do that, as I'm anyway about to rearrange management of the columns names)